source/unified-test-format/unified-test-format.rst

```diff
@@ -225,6 +238,9 @@ Test runners MUST support the following types of entities:
 - ClientSession. See `entity_session`_ and `Session Operations`_.
 - GridFS Bucket. See `entity_bucket`_ and `Bucket Operations`_.
 - ChangeStream. See `ChangeStream Operations`_.
+- Event list. The event list MUST store BSON documents. The type of the list
```

Comment:
Can you link to the `storeEventsAsEntities` option here so readers are not left wondering how they might come to see event lists stored in the entity map?

Reply:
Done.